src/main/java/slimeknights/tconstruct/library/data/AbstractToolItemModelProvider.java

```diff
@@ -91,6 +91,22 @@ protected void bow(IdAwareObject bow, JsonObject properties, boolean crossbow, S
     }
   }
 
+  /** Creates models for blocking, broken and charged for the given tool */
```

Comment:
nitpick, but worth a quick note on what "charged" means. Maybe just say "fully charged" to be more clear?